exporter/awsemfexporter/metric_translator.go

```diff
@@ -29,6 +31,11 @@ const (
 	prometheusReceiver        = "prometheus"
 	attributeReceiver         = "receiver"
 	fieldPrometheusMetricType = "prom_metric_type"
+
+	// metric attributes for AWS EMF, not to be treated as metric labels
+	// emfAttributeFilterRE          = regexp.MustCompile("^awsemf.[A-Za-z_]*$")
+	emfAttributeFilter            = "^awsemf.[A-Za-z_]*$"
```

Comment:
q: why is regex rule is `^awsemf.[A-Za-z_]*$` instead of `^aws.emf.[A-Za-z_]*$`?

Reply:
Thank you @mxiamxia - I've swapped the more complex regex for a simpler comparison operator. As new metric attributes are added this can be expanded to a switch statement which will likely still perform better than a compiled regex.